# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/55_metamail-2.7-fixpartial.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- mm2.7/src/bin/showpartial.fixpartial	Thu Feb  1 12:20:37 2001
+++ mm2.7/src/bin/showpartial	Thu Feb  1 12:21:29 2001
@@ -13,7 +13,7 @@
 endif
 set file="$1"
 # This next line is because message-id can contain weird chars
-set id=`echo $2 | tr -d  \!\$\&\*\(\)\|\'\"\;\/\<\>\\` 
+set id=`echo $2 | tr -d  \!\$\&\*\(\)\|\'\"\;\/\<\>\\\\` 
 @ partnum = "$3"
 if ($#argv == 3 || "$4" == "") then
     set totalnum=-1
@@ -74,9 +74,9 @@
 end
 if ("$found" == "$totalnum") then
     cd "${TREEROOT}/$id"
-    cat "$list" > "${TREEROOT}/$id/FULL"
+    cat $list > "${TREEROOT}/$id/FULL"
 #    cat ${TREEROOT}/$id/HDRS $list > ${TREEROOT}/$id/FULL
-    rm "$list"
+    rm $list
     echo All parts of this "${totalnum}"-part message have now been read.
     metamail -d "${TREEROOT}/$id/FULL"
     echo WARNING:  To save space, the full file is now being deleted.  
